corehq/apps/case_importer/tests/test_importer.py

```diff
@@ -371,6 +372,70 @@ def testParentCase(self):
                          len(res['errors'][exceptions.InvalidParentId.title][error_column_name]['rows']),
                          "All cases should have missing parent")
 
+    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
```

Comment:
Can you comment on this, the fact that we can't run this test on all backends because doing a case import with a domain column doesn't work properly in couch domains?